ReleaseProcessAutomation/Extensions/SemanticVersionExtensions.cs

```diff
@@ -194,6 +226,15 @@ public static SemanticVersion GetNextPatchVersion (this SemanticVersion semantic
     return nextPatchVersion;
   }
 
+  private static IReadOnlyCollection<SemanticVersion> ReleaseRcOrCurrent (SemanticVersion semanticVersion)
+  {
+    return new[]
+           {
+               GetNextRc(semanticVersion),
```

Comment:
Casing: When there's a two-letter acronym, it's uppercase. Either way though, the current code base is inconsistent. We should either do all-upper-case naming for `RC` or make the code base use lower-case if that's a) what we want and b) is less effort because there's way less upper-case spellings in the code.

Reply:
Created a task for it